eventmesh-connectors/eventmesh-connector-file/src/main/java/org/apache/eventmesh/connector/file/source/connector/FileSourceConnector.java

```diff
@@ -73,12 +95,28 @@ public String name() {
 
     @Override
     public void stop() {
-
+        try {
+            if (bufferedReader != null) {
+                bufferedReader.close();
+            }
+            inputStream.close();
+        } catch (Exception e) {
```

Comment:
The `inputStream` has already been closed by `bufferedReader.close()` (according to [JavaDoc](https://docs.oracle.com/javase/8/docs/api/java/io/BufferedReader.html#close--)).

Reply:
Updated